ee/clickhouse/util.py

```diff
@@ -26,17 +27,28 @@ def assertNumQueries(self, num, func=None, *args, using=DEFAULT_DB_ALIAS, **kwar
 
     @contextmanager
     def capture_select_queries(self):
-        from ee.clickhouse.client import _annotate_tagged_query
+        queries = []
```

Comment:
Q: Why does this need a refactor?

Reply:
The specific thing I'm testing for is what is sent to clickhouse. Previously with capture_select_queries we were just looking at _annotate_tagged_query, which isn't the boundary to clickhouse.

I can revert I think, although it does seem a little brittle how it was before.

Reply:
It's okay, was just surprised this needed a refactor.